docs/deprecated.md

```diff
@@ -266,7 +266,7 @@ of the `--changes` flag that allows to pass `Dockerfile` commands.
 
 ### Interacting with V1 registries
 
-**Disabled By Default In Release: v17.04**
+**Disabled By Default In Release: v17.05**
```

Comment:
Wondering if we should only deprecate in _quarterly_ releases, i.e. move this to 17.06? If so we should update the deprecation policy accordingly. We can _mark_ features deprecated in monthly releases, but _remove_ them only in quarterly releases wdyt?